packages/edit-navigation/src/filters/remove-edit-unsupported-features.js

```diff
@@ -10,14 +10,7 @@ const removeNavigationBlockEditUnsupportedFeatures = createHigherOrderComponent(
 			return <BlockEdit { ...props } />;
 		}
 
-		return (
-			<BlockEdit
-				{ ...props }
-				hasSubmenuIndicatorSetting={ false }
-				hasItemJustificationControls={ false }
-				hasColorSettings={ false }
-			/>
-		);
+		return <BlockEdit { ...props } />;
```

Comment:
Look ma, no props! 🥳

Reply:
Nice! It should be possible to delete the whole file and remove the lines where `removeNavigationBlockEditUnsupportedFeatures` is imported and called.

Reply:
🤞